contracts/staking/DonationsStaking.sol

```diff
@@ -7,12 +7,14 @@ pragma experimental ABIEncoderV2;
 import "../Interfaces.sol";
 import "./SimpleStaking.sol";
 import "../utils/DAOUpgradeableContract.sol";
+import "./UniswapV2SwapHelper.sol";
 
 /**
  * @title DonationStaking contract that receives funds in ETH/StakingToken
  * and stake them in the SimpleStaking contract
  */
 contract DonationsStaking is DAOUpgradeableContract {
+	using UniswapV2SwapHelper for SimpleStaking;
```

Comment:
it should be `for this`
uniswapv2swaphelper doesnt really need SimpleStaking. the only method it requires is getRouter, that means
SimpleStaking should be changed to IHasRouter an interface with single getRouter function.
then DonationsStaking need to implement IHasRouter